### PR TITLE
Rs/wx/is hvac sys heating type fluid loop

### PR DESCRIPTION
**Pull request by weilixu**

### CODE REVIEW

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_heating_type_fluid_loop.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    is_hvac_sys_heating_type_fluid_loop_flag = False
+
+    hvac_b = find_exactly_one_with_field_value(
+        "$.buildings[*].building_segments[*].heating_ventilation_air_conditioning_systems",
+        "id",
+        hvac_b_id,
+        rmi_b,
+    )
+    heating_system = hvac_b.get("heating_system")
+    if heating_system:
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_heating_type_fluid_loop.py:34]: This is correct, but could be shortened to:
```
is_hvac_sys_heating_type_fluid_loop_flag = 
    heating_system and
    heating_system.get("hot_water_loop") and
    getattr_(heating_system, "heating_system", "heating_system_type") == HEATING_SYSTEM.FLUID_LOOP
```
And then line 25 could be deleted also.

**Review by charliepnnl — APPROVED** [review]
Approved with one stylistic comment

**Comment by charliepnnl** [comment]
The RDS should also be updated to remove the use of _is_there_only_one_heating_system()_ and to remove the "[0]" from _hvac_b.cooling_system[0]_.

**Review by yunjoonjung-PNNL — APPROVED** [review]
This PR looks great to me! Thank you for this effort.

**Pull request merged**